src/SmartComponents/modules/reducers.js

```diff
@@ -22,47 +79,45 @@ function selectedSystems(selectedIds, selectedSystem) {
 }
 
 function compareReducer(state = initialState, action) {
+    const { payload } = action;
```

Comment:
What's the benefit of saving this off ot a variable vs. calling `action.payload` like we do elsewhere?

Reply:
I must have had it written before in a way that required it to access page and perPage of the payload. I wasn't about to access it via action.payload.page. Now I can!